CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # DrawerKit
 
+## v. 0.7.0
+
+- DrawerKit now supports interactions with presenting view. By default presenting view will receive touch events when drawer is in `collapsed` or `partiallyExpanded` state. This can be configured via new `passthroughTouchesInStates` configuration.
+
+- Drawer corner radius animation can now be disabled.
+
+- Drawer has new `dismissed` state in which the presented view controller will be dismissed. 
```

Comment:
DrawerKit now has a new state, `dismissed`, to differentiate between being `collapsed` (to a particular non-zero height) and being actually dismissed.